.dagger/main.go

```diff
@@ -201,6 +201,19 @@ func (dev *DaggerDev) Test() *Test {
 	return &Test{Dagger: dev}
 }
 
+func (dev *DaggerDev) Generate(ctx context.Context) (*dagger.Directory, error) {
```

Comment:
open question: do we want this to do `dagger call engine generate` as well? under what circumstances does one need to go generate in this codebase?

Reply:
Yeaaah, probably - essentially `engine generate` is just doing `go generate` for all the engine files, which is a lot of `protoc` and `sqlc` things.

Up to you whether to add it here - it doesn't need updating on every api change.

Reply:
added